src/main/java/com/nfl/glitr/util/QueryComplexityCalculator.java

```diff
@@ -267,25 +288,37 @@ public boolean scoreLimitExceeded(String query) {
      * @return query score
      */
     public int queryScore(String query) {
-        return queryScore(parseRootNode(query), 0);
+        String parent = null;
+        if (isMutation(query)) {
+            parent = getMutationName(query);
+        }
+
+        return queryScore(parent, parseRootNode(query), 0);
+    }
+
+    private String getMutationName(String query) {
+        return trimToNull(substringBetween(query, "{", "("));
     }
 
-    private int queryScore(Node queryNode, int depth) {
+    private int queryScore(String path, Node queryNode, int depth) {
         int score = 0;
         int multiplier = 0;
 
         String nodeType = queryNode.getClass().getSimpleName().toUpperCase();
         if (nodeType.equals(QUERY_FIELD) && !isLeaf(queryNode)) {
             Field field = (Field) queryNode;
-            multiplier = extractMultiplierFromListField(field);
+            if (!field.getName().equals("edges") && !field.getName().equals("node")) {
```

Comment:
I don't like having these hardcoded like this. what if an object has a field named "node", I don't believe that it's a reserved word, so I think its conceivable. I wonder if there's a way to determine when they are related to pagination and when they are not?